src/EditorFeatures/CSharpTest/UseExpressionBody/Analyzer/UseExpressionBodyForLocalFunctionsAnalyzerTests.cs

```diff
@@ -0,0 +1,593 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.UseExpressionBody;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseExpressionBody
+{
+    [WorkItem(31477, "https://github.com/dotnet/roslyn/issues/31477")]
+    public class UseExpressionBodyForLocalFunctionsAnalyzerTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
1. were these tests effectively copied from the other method tests?
2. do you have tests for 'when multiline'?

Reply:
1. Most of them effectively were. Since they were very complete already, I thought it'd be a good idea to use them as the basis for these tests.
2. I'll commit them in a moment.